modules/ROOT/pages/install-sys-reqs.adoc

```diff
@@ -95,7 +95,9 @@ To use Runtime Fabric in a production environments, you must also have the follo
 
 * An external load balancer: required if incoming traffic is enabled to Runtime Fabric. A TCP
```

Comment:
GLOBAL for later: 

Suggest reformatting all bullet list items so that the bullet is next to an opening fragment or sentence, followed by a plain para:

     * An external load balancer
        Required if incoming....

     *At least one additional....
       If...when:
            **Performing
            **Performing

(not convinced we need subbullets for just two items....)

Reply:
Ok - thank you!